### PR TITLE
[BACKLOG-40063] - VFS Connection Manager : Allow permission to be applied on the VFS connections created from PUC

### PR DESCRIPTION
**Pull request by soagarwal1**
To be merged after https://github.com/pentaho/pentaho-kettle/pull/9283

Part of PR group

https://github.com/pentaho/pdi-plugins-ee/pull/1050
https://github.com/pentaho/pentaho-kettle/pull/9283
https://github.com/pentaho/metastore/pull/89

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 1m 37s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl s3-vfs
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 1m 30s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl s3-vfs
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

File: s3-vfs/src/main/java/org/pentaho/amazon/s3/provider/S3Provider.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
  *
  * Pentaho Big Data
  *
- * Copyright (C) 2002-2023 by Hitachi Vantara : http://www.pentaho.com
+ * Copyright (C) 2002-2022 by Hitachi Vantara : http://www.pentaho.com
```
Comment (singletonc) [inline s3-vfs/src/main/java/org/pentaho/amazon/s3/provider/S3Provider.java:5]: I don't think we would decrement the copyright year unless this is a reverse commit restoring a previous state?

Comment (soagarwal1) [reply s3-vfs/src/main/java/org/pentaho/amazon/s3/provider/S3Provider.java]: fixed

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 16m 38s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl s3-vfs
```
---
:ok_hand: All tests passed!
 
Tests run: **80**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/big-data-plugin/job/PR-2523/6/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by singletonc — APPROVED** [review]

**Pull request merged**